tests/src/test/scala/cats/tests/MonoidTests.scala

```diff
@@ -1,22 +1,29 @@
 package cats
 package tests
 
-import org.scalatest._
-
 import cats.functor._
 
-class MonoidTests extends FunSuite {
+class MonoidTests extends CatsSuite {
   {
-    import cats.implicits._
```

Comment:
I'm not sure, it seems like all other transformers do not have or need such a comment.